src/Aspire.Dashboard/Components/Controls/Chart/ChartFilters.razor

```diff
@@ -22,19 +22,25 @@
                                 }
                                 else
                                 {
+                                    var i = 0;
                                     foreach (var item in context.SelectedValues.OrderBy(g => g.Name))
                                     {
-                                        <FluentOverflowItem>
-                                            <FluentBadge>@item.Name</FluentBadge>
+                                        <FluentOverflowItem Fixed="@((i == 0) ? OverflowItemFixed.Ellipsis : OverflowItemFixed.None)">
```

Comment:
It's not clear to me why the first item is treated differently. Can we add a comment?

Reply:
Done